scheduler_test.go

```diff
@@ -846,3 +846,19 @@ func TestRunJobsWithLimit(t *testing.T) {
 	assert.Exactly(t, 1, j1Counter)
 	assert.Exactly(t, 1, j2Counter)
 }
+
+func TestErrorDuringJobCreation(t *testing.T) {
+	// error due to bad time format
+	badTime := "0:0"
+	s := NewScheduler(time.Local)
+	s.Every(1).Day().At(badTime).Do(func() {})
```

Comment:
Could we move this test case into the `At` func test?

Reply:
Done